<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateContractPage</class>
 <widget class="QWidget" name="CreateContractPage">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabview">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>create</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Sender</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="create_gasPriceEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>380</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0000004</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>CreateContract</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Constrauctor</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>641</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>ByteCode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="create_byteCodeText"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>390</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Gas Price</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Gas Limit</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>641</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Interface(ABI)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="create_abiText"/>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="create_senderAddress">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>440</y>
       <width>361</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
    <widget class="QPushButton" name="create_btn">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>500</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="create_gasLimitSpinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>40000000</number>
     </property>
     <property name="value">
      <number>2500000</number>
     </property>
    </widget>
    <widget class="QScrollArea" name="create_scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>641</width>
       <height>111</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>639</width>
        <height>109</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>sento</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Sento</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>641</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="sendto_byteCodeText"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>641</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Interface(ABI)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="sendto_abiText"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Function</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sendto_gasLimitSpinBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>420</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>40000000</number>
     </property>
     <property name="value">
      <number>2500000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>430</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Gas Price</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sendto_gasPriceEdit">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>420</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0000004</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Sender</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sendto_senderAddress">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>361</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Gas Limit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sendto_amountText">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>390</y>
       <width>361</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendto_btn">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>530</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Sento</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="sendto_scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>641</width>
       <height>111</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>639</width>
        <height>109</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>call</string>
    </attribute>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Call</string>
     </property>
    </widget>
    <widget class="QPushButton" name="call_btn">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>520</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Call</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="call_scrollArea">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>641</width>
       <height>111</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>639</width>
        <height>109</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>641</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Interface(ABI)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="call_abiText"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>641</width>
       <height>72</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="call_byteCodeText"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Function</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="call_senderAddress">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>390</y>
       <width>361</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SenderAddress</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
